test/e2e/cert_manager_deployment_test.go

```diff
@@ -26,7 +28,8 @@ const (
 var testassets embed.FS
 
 func TestSelfSignedCerts(t *testing.T) {
-	loader := library.NewDynamicResourceLoader(context.TODO(), t)
+	ctx := context.TODO()
```

Comment:
In most of the cases, this should be `context.Background()`. The `context.TODO()` is usually used when there is no context available at hand and you just need to put something into the arguments.